### PR TITLE
ref #327: load pagedef from URL for language toolbar button

### PR DESCRIPTION
**Pull request by pixeljunkie**
| Q             | A
| ------------- | ---
| Branch        | master
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed issues  | chameleon-system/chameleon-system#327
| License       | MIT

### CODE REVIEW

File: src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php [hunk 1]
```diff
@@ -95,10 +96,12 @@ public function GetHTML()
                       </div>';
 
             if ($hasEditPermissionForForeignTable) {
+
                 $url = $urlUtil->getArrayAsUrl(
                     array(
                         'tableid' => $oTargetTableConf->sqlData['id'],
-                        'pagedef' => 'tableeditor', 'id' => $recordId,
+                        'pagedef' => $this->getInputFilterUtil()->getFilteredGetInput('pagedef'),
```
Comment (umulmrum) [inline src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: Could we
- move retrieval of the input filter util before the loop
- add the previous value `tableeditor` as default for `getFilteredGetInput` (or is this wrong in any case?)

Comment (pixeljunkie) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: done

Comment (umulmrum) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: We should then add the default for all such occurrences in this PR.

Comment (pixeljunkie) [reply src/CoreBundle/private/library/classes/TCMSFields/TCMSFieldLookupMultiselectCheckboxes.class.php]: right. Done.

**Comment by umulmrum** [comment]
Thanks @pixeljunkie

**Pull request merged**